src/library/scala/collection/immutable/ListSet.scala

```diff
@@ -45,7 +47,16 @@ object ListSet extends ImmutableSetFactory[ListSet] {
       }
       this
     }
+    /** Discards any existing elements.
+      *
+      * If `result` was called previously, this will reset the builder to build a new instance.
+      */
```

Comment:
The indentation of the `*` is not the same as what I'm used to see in this codebase. Is it intentional? What I usually see is:
```scala
/** First line.
 *
 *  Second line and rest.
 */
```